File: src/algorithms/average-salary.js
/**
 * Given an array of unique integers salary where salary[i] is the salary of the
 * ith employee, returns the average salary of employees excluding the minimum
 * and maximum salary.
 * 
 * @param {number[]} salary - An array of numbers
 * @returns {number} - Returns the mean average excluding min and max numbers
 *     from the array parameter
 */
function average (salary) {
  let sum = 0;
  salary.sort((a, b) => {
    return a - b;
  });
  for (let i = 1; i < salary.length - 1; i++) {
    sum += salary[i];
  }
  return sum / (salary.length - 2);
}

File: src/algorithms/average-salary.py
def average_salary(salary):
    """You are given an array of unique integers salary where salary[i] is the 
    salary of the ith employee.
    Return the average salary of employees excluding the minimum and maximum 
    salary. Answers within 10-5 of the actual answer will be accepted.

    Args:
        salary (list[int]): A list of integers representing salaries.

    Returns:
        float: Returns the average salary excluding min and max salaries.
    """
    total = 0
    salary = sorted(salary)
    for i in range(1, len(salary) - 1):
        total += salary[i]
    return total / (len(salary) - 2)
